engine/src/main/scala/cromwell/engine/workflow/lifecycle/deletion/DeleteWorkflowFilesActor.scala

```diff
@@ -141,9 +157,12 @@ class DeleteWorkflowFilesActor(rootWorkflowId: RootWorkflowId,
   }
 
   whenUnhandled {
+    case Event(ioReceivable, _) if ioReceive.isDefinedAt(ioReceivable) =>
+      ioReceive.apply(ioReceivable)
+      stay
     case Event(ShutdownCommand, _) => stopSelf()
     case other =>
-      log.error(s"Programmer Error: Unexpected message to ${getClass.getSimpleName} ${self.path.name} in state $stateName with $stateData: $other")
+      log.error(s"Programmer Error: Unexpected message to ${getClass.getSimpleName} ${self.path.name} in state $stateName with $stateData: ${other.toString.take(300)}")
```

Comment:
ToL: In this case that surfaced this issue, `other.toString` is up to 12,875,671 characters long, then we're telling scala to take only the first 300 characters.

Reply:
Right - 300 might be too low, but I would say that 12 million is almost certainly too high?

Reply:
https://github.com/broadinstitute/cromwell/blob/develop/common/src/main/scala/common/util/StringUtil.scala#L48
```suggestion
      log.error(s"Programmer Error: Unexpected message to ${getClass.getSimpleName} ${self.path.name} in state $stateName with $stateData: ${other.elided(300)}")
```

Reply:
I was thinking about NOT wasting CPU and memory to generate a 12MB string. Something like [`PPrint`](https://www.lihaoyi.com/PPrint/#Streaming) does. I thought we have something like that already in parts of Cromwell, besides `elided`.